### PR TITLE
Added python3 to interpreter names

### PR DESCRIPTION
**Pull request by v-jizhang**
Added python3 to interpreter names and the search order should be "python3", "python", "python2".

### CODE REVIEW

**Comment by v-jizhang** [comment]
> Just hit this issue yesterday!

There are some test failures. working on it.

File: test/com/facebook/buck/features/rust/RustLibraryIntegrationTest.java [hunk 1]
```diff
@@ -96,7 +97,8 @@ public void rustLibraryCheckWarning() throws IOException {
                 "rust.rustc_check_flags=-Dwarnings --cfg \"feature=\\\"warning\\\"\"",
                 "//messenger:messenger#check")
             .getStderr(),
-        containsString("error: method is never used: `unused`"));
+        either(containsString("error: method is never used: `unused`"))
+            .or(containsString("error: associated function is never used: `unused`")));
```
Comment (zpao) [inline test/com/facebook/buck/features/rust/RustLibraryIntegrationTest.java:101]: This file is unrelated right? Let's pull this out.

Comment (v-jizhang) [reply test/com/facebook/buck/features/rust/RustLibraryIntegrationTest.java]: Done. Thanks
